src/Compilers/Core/Portable/Text/SourceText.cs

```diff
@@ -415,7 +415,7 @@ public virtual void Write(TextWriter writer, TextSpan span, CancellationToken ca
             }
         }
 
-        internal ImmutableArray<byte> GetChecksum(bool useDefaultEncodingIfNull = false)
+        public ImmutableArray<byte> GetChecksum(bool useDefaultEncodingIfNull = false)
```

Comment:
I think we should make this one without default value and add an overload GetChecksum() without parameters and make that one public.